tests/components/panasonic_viera/conftest.py

```diff
@@ -0,0 +1,89 @@
+"""Test helpers for Panasonic Viera."""
+
+from unittest.mock import AsyncMock, Mock, patch
+
+import pytest
+
+from homeassistant.components.panasonic_viera.const import (
+    ATTR_FRIENDLY_NAME,
+    ATTR_MANUFACTURER,
+    ATTR_MODEL_NUMBER,
+    ATTR_UDN,
+    CONF_APP_ID,
+    CONF_ENCRYPTION_KEY,
+    CONF_ON_ACTION,
+    DEFAULT_MANUFACTURER,
+    DEFAULT_MODEL_NUMBER,
+    DEFAULT_NAME,
+    DEFAULT_PORT,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+
+MOCK_BASIC_DATA = {
+    CONF_HOST: "0.0.0.0",
+    CONF_NAME: DEFAULT_NAME,
+}
+
+MOCK_CONFIG_DATA = {
+    **MOCK_BASIC_DATA,
+    CONF_PORT: DEFAULT_PORT,
+    CONF_ON_ACTION: None,
+}
+
+MOCK_ENCRYPTION_DATA = {
+    CONF_APP_ID: "mock-app-id",
+    CONF_ENCRYPTION_KEY: "mock-encryption-key",
+}
+
+MOCK_DEVICE_INFO = {
+    ATTR_FRIENDLY_NAME: DEFAULT_NAME,
+    ATTR_MANUFACTURER: DEFAULT_MANUFACTURER,
+    ATTR_MODEL_NUMBER: DEFAULT_MODEL_NUMBER,
+    ATTR_UDN: "mock-unique-id",
+}
+
+
+def get_mock_remote_entity(device_info=MOCK_DEVICE_INFO):
```

Comment:
This is not the entity, but the device representation.

I think we can remove this function and just patch the `RemoteControl` class. See below.

Yo set the device info we set the return value of the corresponding method of the patched `RemoteControl` instance.